lib/jira_api/jira_issue.py

```diff
@@ -1,11 +1,62 @@
-from typing import Optional
+from typing import Optional, List
 from exceptions.missing_mandatory_param_error import MissingMandatoryParamError
 from exceptions.jira import MismatchedState, ForbiddenTransition
 from structs.jira.jira_account import JiraAccount
 from structs.jira.jira_issue_details import IssueDetails
 from jira_api.connection import JiraConnection
 
 
+def search_issues(
+    account: JiraAccount, project_name: str, requirements_list: Optional[List[str]] = []
+):  # pylint: disable=dangerous-default-value
```

Comment:
I think we can remove disabling this pylint warning with something like this
```suggestion
def search_issues(
    account: JiraAccount, project_name: str, requirements_list: Optional[List[str]] = None
): 
    list = requirements_list or []
```

Reply:
done